### PR TITLE
Boston18stuff

### PR DESCRIPTION
**Pull request by FabianAicheler**

### CODE REVIEW

File: Handout/labelfree.tex [hunk 1]
```diff
@@ -255,15 +254,15 @@ \subsubsection{Identification and Quantification}
...
+\noindent As mentioned before, the downstream analysis can be performed using MSstats. In this case an experimental design has to be specified for the OpenMS workflow. The structure of the experimental design used in OpenMS in case of the iPRG dataset is specified in Table~\ref{t:Experimental_design_iPRG}. An explanation of the variables can be found in Table~\ref{t:Experimental_design_exp}. 
 
 \begin{table}[!ht]
 \centering
 \small
-\caption{OpenMS Experimental design for the iPRG2015 dataset. Note: in the original filenames, the first sample had a hyphen "-" as a separator between sample and biological replicate, the rest of the files had an underscore "\_". We corrected this mistake. The common prefix and the file extension was omitted.}
+\caption{OpenMS Experimental design for the iPRG2015 dataset.}
```
Comment (jpfeuffer) [inline Handout/labelfree.tex]: Why was that removed? I thought it would be quite important.

Comment (timosachsenberg) [reply Handout/labelfree.tex]: I changed it in the mzML files on the stick

Comment (timosachsenberg) [reply Handout/labelfree.tex]: thought it might confuse reader more than it helps to clarify things

Comment (jpfeuffer) [reply Handout/labelfree.tex]: ah ok. but in the experimental design tsv they are still present.
